index.html

```diff
@@ -41,24 +41,6 @@ <h2 class="h3 mb-3 fw-bold text-stride">Accelerated development</h2>
                 </div>
             </div>
         </div>
-        <div class="col-lg-6">
-            <div class="card h-100">
-                <img src="/images/top/platforms.webp" alt="Robot" class="img-fluid p-3">
-                <div class="card-body pt-5 px-5">
-                    <h2 class="h3 mb-3 fw-bold text-stride">Develop once, deploy <br /> cross-platform</h2>
```

Comment:
Should we keep this section and make it like so?

```
                        {% include bullet-point.html text:'Windows Desktop' %}
                        {% include bullet-point.html text:'iOS and Android - experimental state' %}
                        {% include bullet-point.html text:'Linux - experimental state' %}
                        {% include bullet-point.html text:'macOS - experimental state' %}
```

Reply:
To me, we can't really advertise `Develop once, deploy cross-platform` when only one platform actually works. Nothing comes to mind right now, but if you can think of a way to keep that section while being honest about our capabilities, please do let me know and I'll update accordingly

Reply:
That is true. We should be crystal clear with such important information. All good then!